.travis.yml

```diff
@@ -30,6 +30,7 @@ branches:
     only:
         - master
         - 2.7-release
+        - 3.0-release
```

Comment:
You can use a regex here :) e.g. `- /^\d.\d+-release$/`

Reply:
Changed in 3ceac7eaf692bfea54ab11786a172930a3b03203 👍